### PR TITLE
Motion_change_recommendation.create: Fix the collide check.

### PR DESCRIPTION
**Pull request by reiterl**
Two closed intervals are disjunct, iff they don't "touch" each other.
Updated tests and code.

### CODE REVIEW

**Review by emanuelschuetze — CHANGES_REQUESTED** [review]
There are still some collisions in my tests which needs to discussed...

File: tests/system/action/motion_change_recommendation/test_create.py [hunk 1]
```diff
@@ -210,3 +214,12 @@ def test_create_change_recommendation_other_motion_not_colliding(self) -> None:
...
         self.assert_status_code(response, 200)
+
+    def test_create_change_recommendation_collide_check(self) -> None:
+        self.create_change_recommendation(2, 5)
+        response = self.cr_request(1, 2)
+        self.assert_status_code(response, 400)
+        self.assertIn(
+            "The recommendation collides with an existing one",
+            response.json["message"],
+        )
```
Comment (jsangmeister) [inline tests/system/action/motion_change_recommendation/test_create.py:225]: This test can be removed again as it is the same as min_overlap_2

**Review by jsangmeister — CHANGES_REQUESTED** [review]
some cleanup, otherwise looks good

File: tests/system/action/motion_change_recommendation/test_create.py (hunk 1 above)
Comment (reiterl) [reply tests/system/action/motion_change_recommendation/test_create.py]: I removed this surplus test.

**Review by jsangmeister — APPROVED** [review]
We should wait for the merge until is the client is updated as well so the migration works correctly and we do not have mixed data.

**Comment by GabrielInTheWorld** [comment]
I think, you can merge this PR and I will check, that the client's changes (OpenSlides/openslides-client#988) will fit.

**Pull request merged**